### ISSUE TITLE
Dataset Requirement

### ISSUE THREAD

**Issue by AbhayNath001** [ikmckenz/target-pred-py#37]
Can you please send the link of the dataset? Actually, we create the file "chembl_25_sqlite.tar.gz" by your code but unable to use this file. So we are requested to give the link or any drive link as csv format so we can easily use it.

We are waiting for your positive response.

Thank You.

**Comment by ikmckenz** [comment]
How far along the steps did you make it? The make dataset script should prepare the csv of input data, then features are created in the next script, then the model is trained after that. Does it fail along the way?

**Comment by AbhayNath001** [comment]
Yes, first we run the Dataset script, but the csv file was not generated.
